Use of environment variables

sighclone opened the pull request:
Hi! this is for issue #2. I have created a .env file and appropriately use `f'{config["username"]}'` wherever appropriate.<br>
This is my first time doing a no "first time" label issue. Please do check the code once as I am afraid I might have made some mistake.

sighclone committed:
Create .env

sighclone committed:
Update .env

sighclone committed:
Update subscriber.py

sighclone committed:
Update publisher.py

sighclone committed:
Update subscriber.py

sighclone committed:
Update publisher.py

sighclone committed:
Update CONTRIBUTING.md

Santhoshkumard11 commented on subscriber.py:
```diff
@@ -23,6 +24,9 @@ def on_disconnect(client, userdata, rc):
         print("Unexpected disconnection.")
 
+# load the .env values
+config = dotenv_values(".env")
```
@sighclone guess you can move the loading the .env file to the import section of the code as you have done in the published and make sure you have a good commit message - not just update publisher.py 

It's should be something like **updated pusblisher.py -  to import .env file and use the configs for usernames and passwords**

sighclone replied:
Oh okay, I'll do it right now!

sighclone committed:
Update publisher.py to use .env file

updated pusblisher.py - to import .env file and use the configs for usernames and passwords

sighclone committed:
Update subscriber.py to use .env file

updated subscriber.py - to import .env file and use the configs for usernames and passwords

Santhoshkumard11 reviewed (commented):
looking good now

sighclone committed:
Updated publisher.py to import .env file

updated subscriber.py - to import .env file and use the configs for usernames and passwords

sighclone commented:
Yupp I've made the changes

Santhoshkumard11 reviewed (commented):
Thanks for making the change.
